cmd/noms/diff/summary.go

```diff
@@ -84,7 +84,6 @@ func diffSummaryList(ch chan<- diffSummaryProgress, v1, v2 types.List) {
 	spliceChan := make(chan types.Splice)
 	stopChan := make(chan struct{}, 1) // buffer size of 1, so this won't block if diff already finished
 
-	defer stop(stopChan)
```

Comment:
The summary algo doesn't actually stop, so there is no point sending to the stop channel.